src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineIntegrationTest.php

```diff
@@ -66,9 +66,9 @@ public function testSendWithDelay()
             ->setParameter('body', '{"message": "Hi i am delayed"}')
             ->execute();
 
-        $available_at = new \DateTime($stmt instanceof Result || $stmt instanceof DriverResult ? $stmt->fetchOne() : $stmt->fetchColumn());
+        $available_at = new \DateTimeImmutable($stmt instanceof Result || $stmt instanceof DriverResult ? $stmt->fetchOne() : $stmt->fetchColumn());
 
-        $now = new \DateTime();
+        $now = new \DateTimeImmutable();
         $now->modify('+60 seconds');
```

Comment:
This is broken. You need to assign the return value when using DateTimeImmutable, as `->modify` does not mutate it.